packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -23,6 +24,8 @@ interface FixtureModule {
 
 vi.setConfig({ testTimeout: 10_000 /* 10 seconds */ });
 
+const SSR_MODE: CompilationMode = 'sync';
```

Comment:
```suggestion
const SSR_MODE: CompilationMode = 'asyncYield';
```

I think for now this format is easier to read when we inspect the `compiled-experimental-ssr.js` files.